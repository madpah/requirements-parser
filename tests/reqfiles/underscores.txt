a_b>=1.0
