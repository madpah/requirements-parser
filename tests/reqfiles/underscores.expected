[
 {
  "specifier": true,
  "local_file": false,
  "name": "a_b",
  "editable": false,
  "uri": null,
  "extras": [],
  "vcs": null,
  "path": null,
  "line": "a_b>=1.0",
  "specs": [
   [
    ">=",
    "1.0"
   ]
  ],
  "revision": null
 }
]
